subprojects/core/src/test/groovy/org/gradle/api/internal/file/collections/jdk7/Jdk7DirectoryWalkerTest.groovy

```diff
@@ -319,4 +321,30 @@ class Jdk7DirectoryWalkerTest extends Specification {
         }
     }
 
+    @Requires(TestPrecondition.FILE_PERMISSIONS)
+    @Issue('https://github.com/gradle/gradle/issues/2639')
+    @Ignore
```

Comment:
:x: I think we do not want to Ignore the test :)

Reply:
Sure. Removed.